watch.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
Actually I'm starting to think this file does not belong here.

- Still being discussed whether fswatch should be used or not
- This script belongs outside of this repo, a general helper script for Devs (perhaps Daml Connect? or some community Daml goodies)
- In this form the user can easily call 1-liner in Bash (`while true; do sleep 1; make -j; done`)
- The general audience does not need it (trying out, or presenting a demo).